### PR TITLE
Update french i18n + (local) Terminology for jobs

### PR DESCRIPTION
**Pull request by Wadeck**
No associated ticket.

Effort around terminology, in files touched by other corrections in this PR, esp. jobs => projects.

Correction of esclave (and variants) to agents (esclave = slave in french)

Consistency in node configuration page between the dropdown options and the help text.

ℹ️ This PR is a just a limited step toward the perfect situation, not intended to be the whole solution.

### Proposed changelog entries

* Cleanup "esclave" references and fix terminology for jobs in the French localization

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [n/a] (If applicable) Jira issue is well described
- [n/a] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by markjacksonfishing — APPROVED** [review]
I did a google translate but would prefer a native French speaker also review

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/help-slaveAgentPort_fr.html [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ﻿<div>
-  Jenkins utilise un port TCP pour communiquer avec les agents-esclaves
-  lancés par JNLP. Normalement, ce numéro de port est choisi au hasard pour
+  Si vous n'utilisez pas d'agent à connexions entrantes, il est recommendé de
```
Comment (Vlatombe) [inline core/src/main/resources/hudson/security/GlobalSecurityConfiguration/help-slaveAgentPort_fr.html:2]:
```suggestion
  Si vous n'utilisez pas d'agent à connexions entrantes, il est recommandé de
```

File: core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties [hunk 2]
```diff
@@ -20,6 +20,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-\#\ of\ executors=Nb d''ex�cuteurs
+\#\ of\ executors=Nombre d''ex�cuteurs
```
Comment (Vlatombe) [inline core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties:23]:
```suggestion
\#\ of\ executors=Nombre d''ex\u00E9cuteurs
```

File: core/src/main/resources/hudson/slaves/DumbSlave/configure-entries_fr.properties [hunk 3]
```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-\#\ of\ executors=Nb d''ex�cuteurs
+\#\ of\ executors=Nombre d''ex�cuteurs
```
Comment (alecharp) [inline core/src/main/resources/hudson/slaves/DumbSlave/configure-entries_fr.properties:23]:
```suggestion
\#\ of\ executors=Nombre d''ex\u00e9cuteurs
```

**Review by alecharp — COMMENTED** [review]
Missed encoded character?

**Comment by Wadeck** [comment]
IIRC there is no need for encoding characters when we are not using the {0} (variable substitution) anywhere in the properties file.

File: core/src/main/resources/hudson/model/Messages_fr.properties [hunk 4]
```diff
@@ -158,14 +158,14 @@ RunParameterDefinition.DisplayName=Param\u00e8tre d''ex\u00e9cution
...
-Node.Mode.EXCLUSIVE=R\u00e9server cette machine pour les jobs qui lui sont attach\u00e9s seulement
+Node.Mode.EXCLUSIVE=R\u00e9server ce noeud uniquement pour les projets qui lui sont attach\u00e9s
 
 ListView.DisplayName=Vue liste
 
 MyView.DisplayName=Ma vue
 
-LoadStatistics.Legends.TotalExecutors=Nb total d''ex\u00e9cuteurs
-LoadStatistics.Legends.BusyExecutors=Nb d''ex\u00e9cuteurs occup\u00e9s
+LoadStatistics.Legends.TotalExecutors=Nombre total d''ex\u00e9cuteurs
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/model/Messages_fr.properties:167]: Note that the columns are supposed to be short. Better to double-check the UI

File: core/src/main/resources/hudson/slaves/SimpleScheduledRetentionStrategy/config.jelly [hunk 5]
```diff
@@ -36,6 +36,6 @@ THE SOFTWARE.
                checkMessage="${%Scheduled Uptime is mandatory and must be a number.}"/>
     </f:entry>
     <f:entry>
-        <f:checkbox title="${%Keep online while jobs are running}" help="/help/system-config/master-slave/demand/keepUpWhenActive.html" name="retentionStrategy.keepUpWhenActive" checked="${h.defaultToTrue(instance.keepUpWhenActive)}" />
+        <f:checkbox title="${%Keep online while builds are running}" help="/help/system-config/master-slave/demand/keepUpWhenActive.html" name="retentionStrategy.keepUpWhenActive" checked="${h.defaultToTrue(instance.keepUpWhenActive)}" />
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/slaves/SimpleScheduledRetentionStrategy/config.jelly:39]: Not sure we want to derail localizations just by that. Better to add an English-only override (will look like a hack, but will do the job)

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
French changes look good to me. Regarding the terminology changes, I think we need a formal decision in https://github.com/jenkins-infra/jenkins.io/issues/2923 before doing changes back and forth.

A while ago I suggested that the Docs SIG takes lead on finalizing the terminology decisions, but AFAICT the discussion has not happened (CC @MarkEWaite ). Maybe a dev list is a good venue for that.

File: core/src/main/resources/hudson/model/Messages_fr.properties (hunk 4 above)
Comment (Wadeck) [reply core/src/main/resources/hudson/model/Messages_fr.properties:167]: <details>

<summary>Screenshot</summary>

### Load statistics

Screenshot-2020-10-18_11-58-15


</details>

AFAICT it's not this one that is used as a column label.

File: core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties [hunk 6]
```diff
@@ -20,6 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-\#\ of\ executors=Nb d''ex�cuteurs
-Labels=Libell�s
-
+\#\ of\ executors=Nombre d''ex\u00E9cuteurs
+Labels=Libellés
```
Comment (aHenryJard) [inline core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties:24]:
```suggestion
Labels=Libell\u00E9s
```

File: war/src/main/webapp/help/system-config/master-slave/availability_fr.html [hunk 7]
```diff
@@ -9,25 +9,27 @@
...
-            Si Jenkins peut démarrer l'agent sans l'aide de l'utilisateur et
-            que l'agent n'est pas disponible dans la fenêtre où cet
-            il est censé être activé,
-            il tentera régulièrement de le relancer.
-
-            Pendant la fenêtre où l'agent est censé être désactivé, il ne le sera
-            réellement que s'il n'a pas de job en cours.
-        </dd-->
+            <p>
+            Si l'agent n'est pas disponible dans la fenêtre où il est censé être activé,
```
Comment (aHenryJard) [inline war/src/main/webapp/help/system-config/master-slave/availability_fr.html:25]: J'ai du mal avec la fenêtre là, c'était la fenêtre de temps peut être ??

```suggestion
            Si l'agent n'est pas disponible dans au moment où il est censé être activé,
```

File: core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties (hunk 6 above)
Comment (Wadeck) [reply core/src/main/resources/jenkins/model/MasterBuildConfiguration/config_fr.properties:24]: > IIRC there is no need for encoding characters when we are not using the {0} (variable substitution) anywhere in the properties file.

:)

File: war/src/main/webapp/help/system-config/master-slave/availability_fr.html [hunk 8]
```diff
@@ -37,15 +39,15 @@
             cherchera régulièrement à l'activer tant qu'il y aura des jobs en attente
             respectant les critères suivants&nbsp;:
             <ul>
-                <li>Ils sont dans la file d'attente depuis au moins aussi longue que la période d'attente spécifiée</li>
+                <li>Ils sont dans la file d'attente depuis au moins aussi longue que le <i>Délai d'attente lors d'une demande</i> spécifié</li>
```
Comment (aHenryJard) [inline war/src/main/webapp/help/system-config/master-slave/availability_fr.html:42]:
```suggestion
                <li>Ils sont dans la file d'attente depuis au moins aussi longtemps que le <i>Délai d'attente lors d'une demande</i> spécifié</li>
```

**Review by aHenryJard — APPROVED** [review]
Looks good :)

**Review by oleg-nenashev — APPROVED** [review]
LGTM, thanks for addressing the comments!

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

P.S: If this is a Hacktoberfest pull request, please let us know so that we can label it properly and facilitate reviews. Check out [this page](https://www.jenkins.io/events/hacktoberfest/) for more information about Jenkins in Hacktoberfest: guidelines, featured projects, etc. 

[image](https://www.jenkins.io/events/hacktoberfest/)

**Pull request merged**